.github/workflows/build.yml

```diff
@@ -54,35 +58,34 @@ jobs:
         uses: actions/checkout@v3
       - name: Setup
         run: |
-          autoreconf -fiv
-          CC=${{ matrix.compiler }} ./configure
+          CC=${{ matrix.compiler }} meson setup _build
```

Comment:
I really dislike that meson forces you to create a build directory, however if it is necessary can we name it "builddir" as is customary (afaik) with meson projects?
Why prefix it with "_" anyway ?

Reply:
That is because some projects have `build` directly already. I can change it to `builddir`.